src/server/getters/get-reporting-fees.ts

```diff
@@ -62,6 +101,127 @@ interface RepStakeResults {
+        isFinalized: marketParticipants.initialReporters[i].reportingState === "FINALIZED",
+        crowdsourcers: [],
+        initialReporterAddress: marketParticipants.initialReporters[i].initialReporter,
+      };
+    }
+  }
+  for (i = 0; i < marketParticipants.crowdsourcers.length; i++) {
+    if (marketParticipants.crowdsourcers[i].forking && marketParticipants.crowdsourcers[i].universe === parentUniverse) {
+      let forkedMarketIsEmpty: boolean = true;
+      for (const key in forkedMarket) {
+        if (forkedMarket.hasOwnProperty(key)) {
+          forkedMarketIsEmpty = false;
+          break;
+        }
+      }
+      if (forkedMarketIsEmpty) {
+        forkedMarket = {
+          address: marketParticipants.crowdsourcers[i].marketId,
+          universeAddress: marketParticipants.crowdsourcers[i].universe,
+          isFinalized: marketParticipants.crowdsourcers[i].reportingState === "FINALIZED",
+          crowdsourcers: [{address: marketParticipants.crowdsourcers[i].crowdsourcerId, isForked: marketParticipants.crowdsourcers[i].disavowed ? true : false}],
+          initialReporter: {},
+        };
+      } else {
+        forkedMarket.crowdsourcers.push({address: marketParticipants.crowdsourcers[i].crowdsourcerId, isForked: marketParticipants.crowdsourcers[i].disavowed ? true : false});
+      }
+    } else if (!marketParticipants.crowdsourcers[i].forking) {
+      if (!keyedNonforkedMarkets[marketParticipants.crowdsourcers[i].marketId]) {
+        keyedNonforkedMarkets[marketParticipants.crowdsourcers[i].marketId] = {
+          address: marketParticipants.crowdsourcers[i].marketId,
+          universeAddress: marketParticipants.crowdsourcers[i].universe,
+          crowdsourcersAreDisavowed: marketParticipants.crowdsourcers[i].disavowed ? true : false,
+          isMigrated: !marketParticipants.crowdsourcers[i].needsMigration,
+          isFinalized: marketParticipants.crowdsourcers[i].reportingState === "FINALIZED",
+          crowdsourcers: [marketParticipants.crowdsourcers[i].crowdsourcerId],
+          initialReporterAddress: null,
+        };
+      } else {
+        keyedNonforkedMarkets[marketParticipants.crowdsourcers[i].marketId].crowdsourcersAreDisavowed = marketParticipants.crowdsourcers[i].disavowed ? true : false;
+        keyedNonforkedMarkets[marketParticipants.crowdsourcers[i].marketId].crowdsourcers.push(marketParticipants.crowdsourcers[i].crowdsourcerId);
+      }
+    }
+  }
+  const nonforkedMarkets: Array<NonforkedMarket> = [];
+  for (const key in keyedNonforkedMarkets) {
+    if (keyedNonforkedMarkets.hasOwnProperty(key)) {
+      nonforkedMarkets.push(keyedNonforkedMarkets[key]);
+    }
+  }
+
+  return callback(null, {forkedMarket, nonforkedMarkets});
+}
+
+function getMarkets(db: Knex, reporter: Address, universe: Address, parentUniverse: Address, callback: (err: Error|null, formattedMarketInfo?: FormattedMarketInfo) => void) {
```

Comment:
getMarkets is the same name as the getter used for getMarkets. There's no namespace problems, but it might be nice to use a different name so there's no confusion